tests/Client/UpdaterTest.php

```diff
@@ -1280,65 +1240,36 @@ public function testUnsupportedRepo(): void
 
     /**
      * Tests that exceptions are thrown when a repo is in a rollback attack state.
-     *
-     * @param string $fixtureName
-     *   The fixtures set.
-     * @param \Exception $expectedException
-     *   The expected exception.
-     * @param array $expectedUpdatedVersions
-     *   The expected repo file version after refresh attempt.
-     *
-     * @return void
-     *
-     * @dataProvider providerAttackRepoException
      */
-    public function testAttackRepoException(string $fixtureName, \Exception $expectedException, array $expectedUpdatedVersions): void
+    public function testRollbackAttackDetection(): void
```

Comment:
This test case had only one scenario, and it was the same in both fixture variants. So I converted it to stop using a data provider, and renamed it for clarity in testdox.